build/scripts/installer.xml

```diff
@@ -103,7 +103,7 @@
         </fetch>
     </target>
 
-    <target name="xars" depends="download-xars" description="Scan apps directory and include all .xar files into installer">
+    <target name="xars" description="Scan apps directory and include all .xar files into installer">
```

Comment:
I think this is a good idea in general, however I think there might be a problem...

Unfortunately I think this means the Xars will not be correctly downloaded and included when building the installer from a clean build.

May I instead suggest, that you keep the `depends="download-xars"` dependency and instead add a check to the `download-xars` target to skip doing anything if the Xars are already downloaded.